### PR TITLE
[16.0][ADD] account_journal_restrict_mode

### PR DESCRIPTION
**Pull request by JordiMForgeFlow**
Useful module for systems where posted journal entries should not be able to be modified.

### CODE REVIEW

**Review by LoisRForgeFlow — COMMENTED** [review]
You might need to check if reconciliation process does not get blocked

**Comment by JordiMForgeFlow** [comment]
> You might need to check if reconciliation process does not get blocked

It actually works as the reconciliation only modifies the line of the suspense account, but does not modify the name, date, journal or company of the entry. Therefore no error is raised.

However, Odoo actually hides the setting to lock posted entries for bank and cash journals, so maybe we could only force it for other journal types. WDYT?

**Comment by JordiMForgeFlow** [comment]
@LoisRForgeFlow I have leave it as it is and added a comment on the README

File: account_journal_restrict_mode/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Copyright 2023 ForgeFlow S.L.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "Account Journal Restrict Mode",
+    "summary": "Lock All Posted Entries of Journals.",
+    "version": "16.0.1.0.0",
+    "author": "ForgeFlow S.L., " "Odoo Community Association (OCA)",
```
Comment (MiquelRForgeFlow) [inline account_journal_restrict_mode/__manifest__.py]:
```suggestion
    "author": "ForgeFlow S.L., Odoo Community Association (OCA)",
```

**Review by MiquelRForgeFlow — APPROVED** [review]

**Review by AaronHForgeFlow — APPROVED** [review]
LGTM

**Review by JoanSForgeFlow — APPROVED** [review]
LGTM!

**Pull request merged**